koala/excellib.py

```diff
@@ -99,11 +101,13 @@
     "ROUNDUP",
     "POWER",
     "SQRT",
-    "TODAY"
+    "TODAY",
+    "YEAR",
+    "MONTH",
```

Comment:
Does EOMONTH need to be added here?

Reply:
According to the comment just above:

# A dictionary that maps excel function names onto python equivalents. You should
# only add an entry to this map if the python name is different to the excel name
# (which it may need to be to  prevent conflicts with existing python functions
# with that name, e.g., max).

Therefore I think I shouldn't (but a lot of other names also shouldn't).

Reply:
I think that might comment might be for the `FUNCTION_MAP` value, which is a dictionary, the `IND_FUN` is a list. I have no idea what the purpose of `IND_FUN` is, and it is only used by one function `has_ind_func_parent` which leaves me equally confused as to what it does. In any case, why do you think you need to add YEAR and MONTH, but not EOMONTH?

Reply:
Indeed you are right. I've added EOMONTH as well.